src/SFA.DAS.ApprenticeAccounts.Web/TagHelpers/AspNetCoreSimpleUrlHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Routing;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.TagHelpers
+{
+    public interface ISimpleUrlHelper
```

Comment:
I don't think this is used anywhere in this project.  We can probably delete it.